api/metrics/build.gradle

```diff
@@ -6,6 +6,10 @@ plugins {
     id "ru.vyarus.animalsniffer"
 }
 
+version = "${version}".replaceFirst(/^(\d+)\.(\d+).(\d+)/) { _, major, minor, patch ->
```

Comment:
I've stared a lot but don't think I'm missing it - isn't this just `version += '-alpha'`?

Reply:
not with `-SNAPSHOT` versions, where `-alpha` needs to be inserted in the middle.

Reply:
Got it - thanks